tests/acceptance/generate-test.js

```diff
@@ -145,13 +145,10 @@ describe('Acceptance: ember generate', function () {
   });
 
   it('allows a path to be specified to a blueprint', async function () {
-    await outputFile(
-      'blueprints/http-proxy/files/server/proxies/__name__.js',
-      "import Ember from 'ember';\n" + 'export default Ember.Object.extend({ foo: true });\n'
-    );
-    await generate([path.resolve(`${__dirname}/../../blueprints/http-proxy`), 'foo', 'http://localhost:5000']);
```

Comment:
I'm guessing this also worked without the path-support code because this used an absolute path?